test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1183,6 +1200,26 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
 
     #endregion charset encoding tests
 
+    #region Client Certificate Authentication
+
+    It "Verifies Invoke-WebRequest Certificate Authentication Fails without -Certificate"  {
```

Comment:
@iSazonov  The problem with this test is what the intended result is. IMO, the intended result is that the end server is able to see details about the certificate that is passed. That is why the Kestrel server is set to `ClientCertificateMode.AllowCertificate` instead of `ClientCertificateMode.RequireCertificate`. In the real world, this can be implemented in any number of ways. I wanted the method of testing success or fail of this to be as close as possible to each other with the only different being the presence or lack there of a client certificate.